src/setup.py

```diff
@@ -27,19 +27,23 @@
 
 setup(
     name="covid19",
-    version="0.1",
+    version="0.2",
```

Comment:
@brynmathias Happy with this? Have we settled with a convention?

Reply:
We're not settled on a convention yet, I will need to do so, for the moment this is fine.